.gitignore

```diff
@@ -1,56 +1,33 @@
-*.gem
-*.rbc
-/.config
-/coverage/
-/InstalledFiles
-/pkg/
-/spec/reports/
-/spec/examples.txt
-/test/tmp/
-/test/version_tmp/
-/tmp/
+# See https://help.github.com/articles/ignoring-files for more about ignoring files.
+#
+# If you find yourself ignoring temporary files generated by your text editor
+# or operating system, you probably want to add a global ignore instead:
+#   git config --global core.excludesfile '~/.gitignore_global'
 
-# Used by dotenv library to load environment variables.
-# .env
+# Ignore bundler config.
+/.bundle
 
-# Ignore Byebug command history file.
-.byebug_history
+# Ignore the default SQLite database.
+/db/*.sqlite3
+/db/*.sqlite3-*
 
-## Specific to RubyMotion:
-.dat*
-.repl_history
-build/
-*.bridgesupport
-build-iPhoneOS/
-build-iPhoneSimulator/
+# Ignore all logfiles and tempfiles.
+/log/*
+/tmp/*
+!/log/.keep
+!/tmp/.keep
 
-## Specific to RubyMotion (use of CocoaPods):
-#
-# We recommend against adding the Pods directory to your .gitignore. However
-# you should judge for yourself, the pros and cons are mentioned at:
-# https://guides.cocoapods.org/using/using-cocoapods.html#should-i-check-the-pods-directory-into-source-control
-#
-# vendor/Pods/
+# Ignore pidfiles, but keep the directory.
+/tmp/pids/*
+!/tmp/pids/
+!/tmp/pids/.keep
 
-## Documentation cache and generated files:
-/.yardoc/
-/_yardoc/
-/doc/
-/rdoc/
+# Ignore uploaded files in development.
+/storage/*
+!/storage/.keep
 
-## Environment normalization:
-/.bundle/
-/vendor/bundle
-/lib/bundler/man/
-
-# for a library or gem, you might want to ignore these files since the code is
-# intended to run in multiple environments; otherwise, check them in:
-# Gemfile.lock
-# .ruby-version
-# .ruby-gemset
-
-# unless supporting rvm < 1.11.0 or doing something fancy, ignore this:
-.rvmrc
+/public/assets
+.byebug_history
 
-# Used by RuboCop. Remote config files pulled in from inherit_from directive.
-# .rubocop-https?--*
+# Ignore master key for decrypting credentials and more.
+/config/master.key
```

Comment:
You have to add config/database.yml in this file them you run  `git rm --cached config/database.yml`, That will work

Reply:
![Screenshot from 2021-08-20 16-45-55](https://user-images.githubusercontent.com/87749730/130207023-c806a681-4bf3-4afd-9a73-9d9933a92ca1.png)
still cannot

Reply:
This file already is removed by git cache :D, but you haven't push the changes on github, right?
After change any file, steps:
```
- git add .
- git commit -a -m "your message"
- git push origin branch_name -f # this command to apply changes and update to branch on github
```

Reply:
Do you run `git push origin branch_name -f`?